### PR TITLE
Move stale management to github action

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Deleted the existing stale management action from probot and moved it to github actions.

### CODE REVIEW

File: .github/workflows/stale-issues.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 7
+          days-before-issue-close: 7
+          stale-issue-label: "stale"
+          # reason for closed the issue default value is not_planned
+          close-issue-reason: completed
+          only-labels: "stat:awaiting response from contributor"
+          stale-issue-message: > 
+            This issue is stale because it has been open for 7 days with no activity.
```
Comment (fchollet) [inline .github/workflows/stale-issues.yml:23]: We should 2x that number

File: .github/workflows/stale-issues.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            It will be closed if no further activity occurs. Thank you.
+          close-issue-message: >
+            This issue was closed because it has been inactive for 14 days.
```
Comment (fchollet) [inline .github/workflows/stale-issues.yml:26]: We should 2x that number

**Review by fchollet — COMMENTED** [review]
Thanks for the PR

File: .github/workflows/stale-issues.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          close-issue-reason: completed
+          only-labels: "stat:awaiting response from contributor"
+          stale-issue-message: > 
+            This issue is stale because it has been open for 14 days with no activity.
+            It will be closed if no further activity occurs. Thank you.
+          close-issue-message: >
+            This issue was closed because it has been inactive for 28 days.
+            Please reopen if you'd like to work on this further.
+          days-before-pr-stale: 14
+          days-before-pr-close: 14
```
Comment (fchollet) [inline .github/workflows/stale-issues.yml:29]: 28

File: .github/workflows/stale-issues.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Awaiting response issues
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 14
+          days-before-issue-close: 14
```
Comment (fchollet) [inline .github/workflows/stale-issues.yml:17]: 28

File: .github/workflows/stale-issues.yml (hunk 3 above)
Comment (sachinprasadhs) [reply .github/workflows/stale-issues.yml:29]: It is ideally 28 days, 14 days of inactivity for stale label and another 14 days from stale label to close the issues.

Comment (fchollet) [reply .github/workflows/stale-issues.yml:29]: So the delays add up?

**Review by fchollet — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Review by danik292 — APPROVED** [review]

**Comment by sachinprasadhs** [comment]
Creating the changes internally to handle the copybara import issue.

**Pull request closed without merging**